daisy_workflows/image_import/import_image.sh

```diff
@@ -15,13 +15,17 @@
 
 set -x
 
+BYTES_1GB=1073741824
 URL="http://metadata/computeMetadata/v1/instance"
 DAISY_SOURCE_URL="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/daisy-sources-path)"
 SOURCE_DISK_FILE="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/source_disk_file)"
 SOURCEURL=${SOURCE_DISK_FILE}
 SOURCEBUCKET="$(echo ${SOURCEURL} | awk -F/ '{print $3}')"
 SOURCEPATH="${SOURCEURL#"gs://"}"
+SOURCE_FILE_TYPE="${SOURCEURL##*.}"
+SOURCE_SIZE_BYTES="$(gsutil du ${SOURCEURL} | grep -o '^[0-9]\+')"
 DISKNAME="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/disk_name)"
+SCRATCH_DISK_NAME="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/scratch_disk_name)"
```

Comment:
should all metadata variables be at the top, together?

Reply:
sounds good to me!